docs/ref/checks.txt

```diff
@@ -252,6 +252,8 @@ Model fields
   in Django 5.1. *This check appeared in Django 4.2 and 5.0*.
 * **fields.E907**: ``django.contrib.postgres.fields.CITextField`` is removed
   except for support for historical migrations.
+* **fields.E910**: ``SerialField``\s do not accept null values.
+* **fields.E911**: ``SerialField``\s do not accept default values.
```

Comment:
It looks as though `[EW]9??` have all been used for the deprecation of fields, so these could probably do with a different number? It looks as though `[EW]22?` has been used for `GeneratedField`, so maybe use `E300` and `E301` for these? _(We could use that for fields from `django.contrib.postgres`?)_

Better yet, these seem fairly generic, so can we reuse an existing code? `E910` is almost equivalent to `E110`, but perhaps we can just go for:

```
* **fields.E013**: ``<field>`` does not accept null values.
* **fields.E014**: ``<field>`` does not accept default values.
```

Reply:
Thanks @ngnpope , I was not aware. I agree with your generic solution.